e2e/tests/integrations/integrator/create-integration-after-migration.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from "@playwright/test";
+import { KeyVisibility } from "@app-types/KeyVisibility";
+import { IntegrationType } from "@app-types/IntegrationType";
+import { createIntegrationAsIntegrator } from "./create-integration.js";
+import { giveContactKeyVisibility } from "../admin/give-contact-key-visibility.js";
+
+test("As an integrator with migrated projects the key visibility is v1 when creating a new integration", async ({
+  browser,
+}) => {
+  const adminContext = await browser.newContext({
+    storageState: "playwright/.auth/admin.json",
+  });
+  const adminPage = await adminContext.newPage();
+
+  await giveContactKeyVisibility(
+    adminPage,
+    process.env.E2E_TEST_EMAIL!,
+    KeyVisibility.v1
+  );
+
+  const userContext = await browser.newContext({
+    storageState: "playwright/.auth/user.json",
+  });
+  const userPage = await userContext.newPage();
+
+  const { integrationName } = await createIntegrationAsIntegrator(
+    userPage,
+    IntegrationType.SearchApi
+  );
+  await userPage.waitForURL(/https?:\/\/[^/]*\/nl\/integraties(\/.*)?/);
+  await expect(userPage.getByText(integrationName)).toBeVisible();
+
+  await adminPage.goto("/admin/resources/integrations");
+  await adminPage
```

Comment:
I wonder if it isn't safer to check the key visibility on the detail page instead of on the overview page.
On the overview page there are a lot of integrations (with pagination). 
So it could be possible that there's another integration with the same name which has another keyvisibility...